.npmignore

```diff
@@ -37,3 +37,5 @@ Dockerfile
 .eslintrc.json
 .yarnrc
 tmp
+.eslintplugin.js
+.eslintcache
```

Comment:
I should have included this on my eslint-at-root PR; I really dislike npmignore for this reason. Maybe I'll sit down and change our package to use `"files"` instead.